libs/application/templates/aosh/practical-exam/src/fields/Components/ExamineesCSVUploader.tsx

```
@@ -0,0 +1,371 @@
+                  index,
+                  duplicateEmails,
+                )
+              if (parsedExaminee.nationalId.nationalId)
+                trackDuplicates(
+                  seenNationalIds,
+                  parsedExaminee.nationalId.nationalId,
+                  index,
+                  duplicateNationalIds,
+                )
+
+              return [...validExaminees, parsedExaminee]
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)